src/main/java/com/board/config/boardconfiggui/BoardConfigController.java

```diff
@@ -139,7 +139,7 @@ private void setTreeView() {
     private void generateOutput() {
         saveCurrentControllerData();
         if(Utils.saveData(xmlFolderPath))
-            homeViewController.onOutputGenerateClick();
+            homeViewController.onOutputGenerateClick(xmlFolderPath);
```

Comment:
No need to pass this information back to homeviewController instead you can persist this information globally inside homeviewController